mysql/scripts/config_mariadb.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+__mysql_config() {
+  echo "Running the mysql_config function."
+  mysql_install_db
+  chown -R mysql:mysql /var/lib/mysql
+  /usr/bin/mysqld_safe &
+  sleep 10
+}
+
+__start_mysql() {
```

Comment:
This will probably need be changed to accept inputs or a way to post back to the user:

https://github.com/alexdicianu/argos/blob/master/argos#L93